include/lsst/afw/detection/Psf.h

```diff
@@ -290,6 +290,16 @@ class Psf : public afw::table::io::PersistableFacade<Psf>,
      */
     explicit Psf(bool isFixed = false, std::size_t capacity = 100);
 
+    /**
+     *
+     * This protected virtual member was formerly private (see message below).  We switched it to
+     * protected so that the Psf trampoline class can call it if a python-implemented Psf-derived
+     * class doesn't provide an override.  In contrast, pure virtual private members may remain
+     * private since they must be overridden.
```

Comment:
Rewrite:
```
    /**
     * This virtual member is protected (rather than private) so that python-implemented derived
     * classes may opt to use the default implementation.  C++ derived classes may override this
     * method, but should not call it.
     */
```